src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,31 +1,39 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
-	private int[] numbers = new int[12];
-	public int total = -1;
+// TODO: check name.
+	private static final int FULL_STACK_VALUE = 11;
+	private static final int EMPTY_STACK_VALUE = -1;
+	private static final int STACK_CAPACITY = 12;
+	private int[] numbers = new int[STACK_CAPACITY];
+	private int total = EMPTY_STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)